types.d.ts

```diff
@@ -11634,7 +11634,7 @@ declare interface UserResolveOptions {
 	restrictions?: (string | RegExp)[];
 
 	/**
-	 * Use only the sync constiants of the file system calls
+	 * Use only the sync constraints of the file system calls
```

Comment:
This appears to be an upstream issue in https://github.com/webpack/enhanced-resolve, so i'll submit a PR there